### PR TITLE
implement some basic print styles

### PR DESCRIPTION
**Pull request by michaelficarra**
These follow Ecma's guidelines.

### CODE REVIEW

File: css/print.css [hunk 1]
```diff
@@ -1,3 +1,28 @@
-#menu-toggle {
-  display: none;
```
Comment (bakkot) [inline css/print.css:2]: Why get rid of this?

Comment (michaelficarra) [reply css/print.css:2]: that element wasn't present when using the old TOC

Comment (bakkot) [reply css/print.css:2]: It is not yet the case that "printing" is equivalent to "old TOC".

I guess it's fine landing now but I'd've preferred doing it as part of that.

Comment (bakkot) [reply css/print.css:2]: I guess the new TOC is hidden when printing anyway so it doesn't matter.

**Review by bakkot — APPROVED** [review]

**Pull request merged**